samples/FunctionApp/HttpTriggerWithBlobInput/HttpTriggerWithBlobInput.cs

```diff
@@ -11,6 +11,7 @@ namespace FunctionApp
     public static class HttpTriggerWithBlobInput
     {
         [Function(nameof(HttpTriggerWithBlobInput))]
+        [ExponentialBackoffRetry(5, "00:00:04", "00:15:00")]
```

Comment:
Would retry work with HttpTrigger? I thought it would not. If so, we should remove this example to avoid confusing users.